src/transformers/models/llava/modeling_llava.py

```diff
@@ -321,6 +325,8 @@ def _merge_input_ids_with_image_features(
         # we need to index copy on [0, 577, 578, 579] for the text and [1:576] for the image features
         final_embedding[batch_indices, text_to_overwrite] = inputs_embeds[batch_indices, non_image_indices]
         final_attention_mask[batch_indices, text_to_overwrite] = attention_mask[batch_indices, non_image_indices]
+        if labels is not None:
+            final_labels[batch_indices, text_to_overwrite] = input_ids[batch_indices, non_image_indices]
```

Comment:
```suggestion
        if labels is not None:
            final_labels[batch_indices, text_to_overwrite] = labels[batch_indices, non_image_indices]
```
why don't we use the provided labels?